scripts/zones/Ceizak_Battlegrounds/mobs/Mastop.lua

```diff
@@ -0,0 +1,66 @@
+-----------------------------------
+-- Area: Ceizak Battlegrounds
+-- NM: Mastop
+-----------------------------------
```

Comment:
This will also need comments with commands for `!pos` (take you to the NM's spawn point), and `!additem` or some sort of `!exec` to force-pop the NM, for easier testing